utils_cv/detection/references/anno_coco2voc.py

```diff
@@ -0,0 +1,214 @@
+# Code copied and slightly modified from:
+# https://github.com/CasiaFan/Dataset_to_VOC_converter/blob/master/anno_coco2voc.py
+#
+# Most modifications are hlighlighted by the keyword "EDITED".
```

Comment:
you probably did this, but best to make sure *all* modifications are highlighted with EDITED

Reply:
Let me double-check, but yes, that was my intention